test/unit/app/models/edition_test.rb

```diff
@@ -382,6 +382,11 @@ class EditionTest < ActiveSupport::TestCase
     assert_not build(:edition, summary: nil).valid?
   end
 
+  test "should be be valid without taxons when editionable_worldwide_organisation" do
+    edition = build(:draft_editionable_worldwide_organisation)
+    assert edition.valid?
+  end
```

Comment:
This probably ought to be in the `EditionableWorldwideOrganisationTest`, since it applies only to that model.

Reply:
Moved 👍

Reply:
I think you can remove the test here too, as it's been copied into the other place.